aten/src/ATen/cpu/vec/vec256/vec256_convert.h

```diff
@@ -314,6 +314,23 @@ struct VecConvert<float, 1, BFloat16, 1> {
     return result;
   }
 };
+
+template <>
+struct VecConvert<BFloat16, 1, float, 2> {
+  static inline VectorizedN<BFloat16, 1> apply(
+      const VectorizedN<float, 2>& src) {
+    VectorizedN<BFloat16, 1> result;
+    uint32x4x2_t u32x4x2_0 = vld1q_u32_x2(reinterpret_cast<const uint32_t*>(&src[0]));
+    uint32x4x2_t u32x4x2_1 = vld1q_u32_x2(reinterpret_cast<const uint32_t*>(&src[1]));
+    uint16x4_t u16x4_0 = vmovn_u32(vshrq_n_u32(u32x4x2_0.val[0], 16));
+    uint16x4_t u16x4_1 = vmovn_u32(vshrq_n_u32(u32x4x2_0.val[1], 16));
```

Comment:
```suggestion
    uint16x8_t u16x8_low =vshrn_high_n_u32(u16x4_0, u32x4x2_0.val[1], 16);
```

and this can be used to avoid vcombine